contrib/packaging/bootupd-static-grub-migration.service

```
@@ -0,0 +1,17 @@
+[Unit]
+Description=bootupd static GRUB config migration
+Documentation=https://github.com/coreos/bootupd
+ConditionPathExists=!/boot/.bootupd-static-migration-complete
```

If we don't want to leave another stamp file around we can use
```suggestion
ConditionPathIsSymbolicLink=/etc/grub2/grub.cfg
```

---

It's `/boot/grub2/grub.cfg` that is a symlink but we can not do that as this would skip all the logic after the symlink change that is still needed.